templates/hri-base.html.twig

```diff
@@ -87,7 +87,7 @@
                                 <img src="{{ asset('build/images/hri-logo-min.svg') }}" alt="logo"
                                      class="mw-100 logo">
                             </a>
-                            <a class="navbar-brand" href="https://data.gulfresearchinitiative.org/" target="_blank">
+                            <a class="navbar-brand" href="https://griidc.org" target="_blank">
```

Comment:
Should this be hri.griidc.org here? I'm unsure...PO question.

Reply:
It's the GRIIDC logo, should go to GRIIDC. (.org)